libtrellis/src/Bitstream.cpp

```diff
@@ -732,6 +798,65 @@ Bitstream Bitstream::serialise_chip(const Chip &chip, const map<string, string>
     // Padding
     wr.insert_dummy(ops.dummy_bytes_after_preamble);
 
+    if (chip.info.family == "MachXO") {
```

Comment:
might be neater to split this off into a `serialise_chip_machxo` function?

Reply:
done